gcp/cloud_run_functions/run_followup_scheduling/main.py

```
@@ -76,16 +76,33 @@
+            or None for existing participants who predate the shift question.
+
+    Returns:
+        List of three time objects in the participant's local timezone.
+
+    Raises:
+        RuntimeError: If shift_times is absent from the fn3 config.
+        KeyError: If work_shift is not in config.shift_times.shifts.
+            Pub/Sub will retry -- no SMS is scheduled for bad shift data.
+    """
+    if config.shift_times is None:
+        raise RuntimeError("shift_times missing from fn3 config")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)